lib/Github/Api/Apps.php

```diff
@@ -5,28 +5,14 @@
 use Github\Api\AcceptHeaderTrait;
 
 /**
- * @link   https://developer.github.com/v3/integrations/
+ * @link   https://developer.github.com/v3/apps/
  * @author Nils Adermann <naderman@naderman.de>
  */
-class Integrations extends AbstractApi
+class Apps extends AbstractApi
```

Comment:
Also to keep BC the class can not be renamed (in my opinion). Bu the class can be deprecated and a new Apps class can be created (where this class can extend from)